### ISSUE TITLE
`raiden_chain` fixture setup fail

### ISSUE THREAD

**Issue by pcppcp** [raiden-network/raiden#1704]
## Problem Definition
With some tests and when using Matrix transport, the `raiden_chain` fixture initialization fails. Apparently some nodes won't sync their channel state over the Matrix in time (timeout is 60 seconds).
It's hard to reproduce it locally, but on Travis this happens quite often (probably due to VM constraints).

### Tests skipped because of this bug
`raiden/tests/integration/test_service.py::test_udp_ping_pong`
`raiden/tests/integration/test_service.py:test_udp_ping_pong_unreachable_node:`
`raiden/tests/integration/test_recovery.py::test_recovery_happy_case`
`raiden/tests/integration/test_recovery.py::test_recovery_unhappy_case`
`raiden/tests/integration/test_echo_node.py::test_event_transfer_received_success`
`raiden/tests/integration/transfer/test_refund_transfer.py::test_refund_transfer_after_2nd_hop`

### Details

Example traceback:
```
ERROR at setup of test_event_transfer_received_success[transport_config0-64-18-channels_per_node0-1-4] 
>   lambda: ihook(item=item, **kwds),
    when=when,
            )
../../../virtualenv/python3.6.3/lib/python3.6/site-packages/flaky/flaky_pytest_plugin.py:273: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
raiden/tests/integration/fixtures/raiden_network.py:95: in raiden_chain
    deposit,
raiden/tests/utils/network.py:429: in wait_for_channels
    retry_timeout,
raiden/tests/utils/network.py:392: in wait_for_usable_channel
    retry_timeout,
raiden/waiting.py:91: in wait_for_participant_newbalance
    gevent.sleep(retry_timeout)
../../../virtualenv/python3.6.3/lib/python3.6/site-packages/gevent/hub.py:155: in sleep
    hub.wait(t)
src/gevent/_hub_primitives.py:46: in gevent.__hub_primitives.WaitOperationsGreenlet.wait
    ???
src/gevent/_hub_primitives.py:55: in gevent.__hub_primitives.WaitOperationsGreenlet.wait
    ???
src/gevent/_waiter.py:152: in gevent.__waiter.Waiter.get
    ???
src/gevent/_greenlet_primitives.py:59: in gevent.__greenlet_primitives.SwitchOutGreenletWithLoop.switch
    ???
src/gevent/_greenlet_primitives.py:59: in gevent.__greenlet_primitives.SwitchOutGreenletWithLoop.switch
    ???
src/gevent/_greenlet_primitives.py:63: in gevent.__greenlet_primitives.SwitchOutGreenletWithLoop.switch
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
>   ???
E   RuntimeError: `raiden_chain` fixture setup failed, nodes are unreachable
src/gevent/__greenlet_primitives.pxd:35: RuntimeError
```

**Comment by andrevmatos** [comment]
I can consistently reproduce the traceback locally by reducing the timeout, e.g. to 1s, [here](https://github.com/raiden-network/raiden/blob/863959920d7d29d8f81f1eb9c1ae50c5554a9ddb/raiden/tests/integration/fixtures/raiden_network.py#L90). Unfortunatelly, looks like increasing the timeout isn't enough to fix the issue, as I did in 9cb4def74c20782f619ffc8e4fd5adda07eab8b3 from 30s to 60s and the fixture is still flaky.

**Comment by LefterisJP** [comment]
Added `raiden/tests/integration/transfer/test_refund_transfer.py::test_refund_transfer_after_2nd_hop` to the skipped tests.

**Comment by andrevmatos** [comment]
Fixed by #1708
